lib/datasource/go/index.ts

```diff
@@ -1,11 +1,14 @@
 import type { GetReleasesConfig, ReleaseResult } from '../types';
+import { getDigest as _getDigest } from './digest';
 import * as direct from './releases-direct';
 import * as goproxy from './releases-goproxy';
 
 export { id } from './common';
```

Comment:
Please use this export style

Reply:
I've tried, but it caused coverage to fail with 100% value being reported. This explicit import/export is the first thing I tried, but maybe we have a better solution?

Reply:
Yes, you need to import it from this file in tests to get it covered. 😉

Reply:
Thanks!